soc/wch/ch32v00x/soc.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Michael Hope
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SOC_H__
+#define __SOC_H__
+
+#ifndef _ASMLANGUAGE
+#include <zephyr/types.h>
+#endif /* !_ASMLANGUAGE */
+
+#define SOC_MCAUSE_EXP_MASK          CONFIG_RISCV_SOC_MCAUSE_EXCEPTION_MASK
```

Comment:
I did not found an use for this. Is it really necessary? If that is the case, wouldn't be better just use the CONFIG_RISCV_SOC_MCAUSE_EXCEPTION_MASK insetad?

Reply:
Yes, it is not used. I have removed this line now.